### PR TITLE
Update checkpointctl v0.1.0

### PR DESCRIPTION
**Pull request by adrianreber**
With a minor fix during error unwrapping.

Replaces #17674 as it requires a minor fix during error unwrapping.

CC: @vrothberg

### CODE REVIEW

File: pkg/checkpoint/crutils/checkpoint_restore_utils.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func CRImportCheckpointConfigOnly(destination, input string) error {
 // it exists deletes all files listed.
 func CRRemoveDeletedFiles(id, baseDirectory, containerRootDirectory string) error {
 	deletedFiles, _, err := metadata.ReadContainerCheckpointDeletedFiles(baseDirectory)
-	if os.IsNotExist(errors.Unwrap(errors.Unwrap(err))) {
+	if os.IsNotExist(errors.Unwrap(err)) {
```
Comment (Luap99) [inline pkg/checkpoint/crutils/checkpoint_restore_utils.go:78]: this should just be `errors.Is(err, os.ErrNotExist)` there is no need to manually unwrap.

**Review by Luap99 — COMMENTED** [review]
one nit

File: pkg/checkpoint/crutils/checkpoint_restore_utils.go (hunk 1 above)
Comment (adrianreber) [reply pkg/checkpoint/crutils/checkpoint_restore_utils.go]: removed the unwrapping

**Review by vrothberg — COMMENTED** [review]
Thank you, @adrianreber !

/hold cancel

**Pull request merged**